.github/stale.yml

```diff
@@ -1,25 +1,34 @@
 # Number of days of inactivity before an issue becomes stale
 daysUntilStale: 30
+
 # Number of days of inactivity before a stale issue is closed
 daysUntilClose: 14
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels:
+  - "additional information requested"
+
 # Issues with these labels will never be considered stale
 exemptLabels:
   - backlog
   - epic
   - pinned
   - security
   - triage
+
 # Label to use when marking an issue as stale
-staleLabel: wontfix
+staleLabel: stale
+
 # Comment to post when marking an issue as stale. Set to `false` to disable
 markComment: >
   This issue has been automatically marked as stale because it has not had
   recent activity. We appreciate your contributions, and we understand 
   things get busy.  This issue will be closed after 14 days if no 
-  further activity occurs. 
+  further activity occurs, but can always be reopened at a later date. 
   Thank you for your contributions.
+
 # Comment to post when closing a stale issue. Set to `false` to disable
 closeComment: >
   This issue has been automatically closed because it has not had
-  recent activity.
+  recent activity, but it can be reopened again.
```

Comment:
nit: ... maybe change it to: `it can be reopened` (my preference) or `it can be opened again`.

`it can be reopened again` feels a bit redundant.